app/views/users/posts.html.erb

```diff
@@ -0,0 +1 @@
+<h1> this is a multi posts view</h1>
```

Comment:
- [ ] Also you should create a separate folder `posts` in views and its index and show pages